### PR TITLE
Tools: uploader.py: print board name as well as board id

### PR DESCRIPTION
**Pull request by peterbarker**
This only works from the root directory.  Do people invoke waf elsewhere?

### CODE REVIEW

**Comment by peterbarker** [comment]
Example output (look under "Info" towards the bottom):

```
pbarker@bluebottle:~/rc/ardupilot(pr/board-name)$ ./Tools/scripts/uploader.py --identify
If the board does not respond within 1-2 seconds, unplug and re-plug the USB connector.
Found board 88,0 bootloader rev 5 on /dev/serial/by-id/usb-mRo_X2.1-777_1B0031000351373236383732-if00
Bootloader Protocol: 5
OTP:
  type: ÿÿÿÿ
  idtype: =FF
  vid: ffffffff
  pid: ffffffff
  coa: //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////8=
  sn: 0031001b3237510332373836
ChipDes:
  family: STM32F7[6|7]x
  revision: Z
Chip:
  STM32F76x_77x 10016451
Info:
  flash size: 1998848
  board_type: 136 (mRoX21-777)
  board_rev: 0
Identification complete
pbarker@bluebottle:~/rc/ardupilot(pr/board-name)$ 
```

**Comment by Hwurzburg** [comment]
by root do you mean ardupilot?the directory that contains the clone, that is where I always invoke it...that's where git is tracking for me

**Comment by peterbarker** [comment]
@Hwurzburg yes, I should have said, "root directory of the ArduPilot git repository checkout" rather than relying on context.

File: Tools/scripts/uploader.py [hunk 1]
```diff
@@ -672,11 +673,51 @@ def dump_board_info(self):
...
+        shared_ids = {
+            9: "fmuv3",
+            50: "fmuv5",
+        }
+        if board_id in shared_ids:
+            return shared_ids[board_id]
+
+        try:
+            ret = []
+            hwdef_dir = "libraries/AP_HAL_ChibiOS/hwdef"
```
Comment (tridge) [inline Tools/scripts/uploader.py:696]: why not derive from path of the script?

Comment (peterbarker) [reply Tools/scripts/uploader.py]: Done.

**Pull request merged**